economy.md

```diff
@@ -2,22 +2,16 @@
 
 ## bal
 
-- Quality.
-
-  <img align="top" src="https://user-images.githubusercontent.com/86816930/142962190-9722fd2a-2b7e-4851-a03e-f182e59f3bfc.png">
+- Test the fucking command and you'll find out glory. (you apparentally are incapable of doing this whenever coding!)
```

Comment:
If you could add screenshots of the various bugs that would be great.

Reply:
no